### ISSUE TITLE
Important li!Please ADD these EXISTING features to the description page

### ISSUE THREAD

**Issue by sangos** [kylegilman/video-embed-thumbnail-generator#383]
First off congratulations to the film editor and plugin author extraordinaire! IMO your plugin blows all those other 'also rans' boom out of the water. That said please add these  **already existing features** to the description page @wordpress. Took me some time to figure it out from the long write up. Screenshots are the best. Am sure a lot of folks will be sold on these! Here goes:

A. There is a lightbox to display video. This lightbox has all the bells and whistles..........
B. There are 4 players to choose from. Yes thats correct 4 and one of them gives sick skin options.......
C. Video Quality Switcher aka just like Youtube. You have the options of either uploading or auto encode......

Thanks for your attention.

**Comment by kylegilman** [comment]
All of those things are already on the description page on WordPress with different words than you might have been looking for. I first wrote that description several years ago and have added to it as features were added, and it's not particularly user friendly. I'm in the middle of rewriting the whole plugin, and once that project is finished I'll have a better description and a website dedicated to explaining the features of the plugin.
